test/components/scroll-spy.spec.coffee

```diff
@@ -0,0 +1,61 @@
+{React, Testing, expect, _, sinon} = require './helpers/component-testing'
+
+ScrollSpy = require '../../src/components/scroll-spy'
+
+
+NestedComponent = React.createClass
+  render: -> React.createElement('div')
+
+RootComponent = React.createClass
+  render: ->
+
+    React.createElement('div', {},
+      React.createElement(ScrollSpy, @props,
+        React.createElement(NestedComponent, onScreenElements:[])
+      )
+    )
```

Comment:
Well, how about that. I didn't know were were doing that. I'll redo as cjsx then, that'll make it much clearer what's going on.